### PR TITLE
Refactoring of examples folder

### PR DESCRIPTION
**Pull request by javiergonzalezh**
*Issue #72 , if available:* 

*Description of changes:* I have refactored the examples and the tutorial folders. The idea is to have a clear structure of how to contribute to the library.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: emukit/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Emukit Examples
+
+This page contains a curated list of Emukit examples and links to other tutorials. It is heavily inspired on [contributing
```
Comment (apaleyes) [inline emukit/examples/README.md]:
```suggestion
This page contains a curated list of Emukit examples and links to other tutorials. It is heavily inspired by [contributing
```

File: emukit/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+examples](https://github.com/apache/incubator-mxnet/blob/master/example/README.md) section of [MXnet](https://mxnet.apache.org/).
+ 
+  - [Contributing](#contributing)
+  - [List of examples](#list-of-examples)
+  - [List of tutorials](#list-of-tutorials)
+
+
+## <a name="Contributing"></a>Contributing
+If you want to contribute to this of, please open a new pull request.
```
Comment (apaleyes) [inline emukit/examples/README.md]:
```suggestion
If you want to contribute to this folder, please open a new pull request.
```

File: emukit/examples/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### Examples
+
+Examples can be either notebooks that tell a story about a problem/question using Emukit, *e.g* the analysis of the properties of a simulator, or
+they contain the implementation of some specific method. Examples can live in a .py file, and ideally have tests and
```
Comment (apaleyes) [inline emukit/examples/README.md]:
```suggestion
they can contain an implementation of some specific method. Examples can live in a .py file, and ideally have tests and 
```

File: emukit/examples/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+### Examples
+
+Examples can be either notebooks that tell a story about a problem/question using Emukit, *e.g* the analysis of the properties of a simulator, or
+they contain the implementation of some specific method. Examples can live in a .py file, and ideally have tests and 
+come with an illustrative notebook. 
+
+#### Examples location
+
+Example applications or scripts should be submitted in this `emukit/examples` folder.  Each example must live in a separated 
+folder that can contain some extra files an dependencies. Please make sure that you update this `README.md` file with the information
```
Comment (apaleyes) [inline emukit/examples/README.md]:
```suggestion
folder that can contain some extra files and dependencies. Please make sure that you update this `README.md` file with the information 
```

File: emukit/examples/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Examples can be either notebooks that tell a story about a problem/question using Emukit, *e.g* the analysis of the properties of a simulator, or
+they contain the implementation of some specific method. Examples can live in a .py file, and ideally have tests and 
+come with an illustrative notebook. 
+
+#### Examples location
+
+Example applications or scripts should be submitted in this `emukit/examples` folder.  Each example must live in a separated 
+folder that can contain some extra files an dependencies. Please make sure that you update this `README.md` file with the information 
+about you example when submitting and PR with an example.
```
Comment (apaleyes) [inline emukit/examples/README.md]:
```suggestion
about you example when submitting a PR.
```

File: emukit/examples/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+
+#### Examples tests
+
+As part of making sure all our examples are running correctly with the latest version of Emukit, yor can add your own tests 
+here `emukit/tests/examples/test_example.py`. (If you forget, don't worry your PR will not pass the sanity check).
```
Comment (apaleyes) [inline emukit/examples/README.md]: Not sure about that. If there are no tests, PR will still pass the build alright. But you could say "if you forget, don't worry, we'll remind you during the review"

File: notebooks/index.ipynb [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Emukit tutorials\n",
+    "\n",
+    "Emukit tutorials can be added and used through the links below. The goal on these tutorials is to explain a particular functionality of Emukit. These tutorials are stand alone notebooks that don't require any extra file and fully sit on Emukit components (apart from the creation of the model).\n",
+    "\n",
+    "Some tutorials have been written with the purpose of explaining some scientific concepts and can be used to learning about different topics in emulation and uncertainty quantification. Other tutorials are a small guide to describe some new feature.\n",
```
Comment (apaleyes) [inline notebooks/index.ipynb]:
```suggestion
    "Some tutorials have been written with the purpose of explaining some scientific concepts and can be used for learning about different topics in emulation and uncertainty quantification. Other tutorials are a small guide to describe some feature of the library.\n",
```

or "..as a learning material..."

File: notebooks/index.ipynb [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+    "    * [Bayesian optimization with external evaluation of the objective](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-Bayesian-optimization-external-objective-evaluation.ipynb)\n",
+    "    * [Benchmark automatically several Bayesian optimization methods](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-Bayesian-optimization-benchmark.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **Do it yourself:**\n",
+    "    * [How to write a tutorial model for Emukit](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-how-to-write-a-notebook.ipynb) (please follow the contribution guide) \n",
```
Comment (apaleyes) [inline notebooks/index.ipynb]:
```suggestion
    "    * [How to write a tutorial notebook for Emukit](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-how-to-write-a-notebook.ipynb) (please follow the contribution guide) \n",
```

File: notebooks/index.ipynb [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+    "    * [How to use custom model](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-custom-model.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Contribution guide\n",
+    "\n",
+    "We really appreciate contributions. Tutorials and examples are a great way to spread what you have learned about, Emukit across the community and an excellent way to showcase new features. If you want to contribute with a new tutorial please follow [these](https://github.com/amzn/emukit/tree/develop/emukit/examples) steps.\n",
```
Comment (apaleyes) [inline notebooks/index.ipynb]: "Emukit across the community" - is emukit a verb now?

File: notebooks/index.ipynb [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+    "\n",
+    "We also welcome feedback so if there is any aspect of the Emukit that we can improve please raise an issue!"
```
Comment (apaleyes) [inline notebooks/index.ipynb]:
```suggestion
    "We also welcome feedback so if there is any aspect of Emukit that we can improve please raise an issue!"
```

File: emukit/examples/README.md (hunk 6 above)
Comment (javiergonzalezh) [reply emukit/examples/README.md]: done

File: notebooks/index.ipynb (hunk 9 above)
Comment (javiergonzalezh) [reply notebooks/index.ipynb]: changed

File: notebooks/index.ipynb [hunk 11]
```diff
@@ -65,10 +65,17 @@
...
     "## Contribution guide\n",
     "\n",
-    "We really appreciate contributions. Tutorials and examples are a great way to spread what you have learned about, Emukit across the community and an excellent way to showcase new features. If you want to contribute with a new tutorial please follow [these](https://github.com/amzn/emukit/tree/develop/emukit/examples) steps.\n",
+    "We really appreciate contributions. Tutorials and examples are a great way to spread what you have learned about Emukit across the community and an excellent way to showcase new features. If you want to contribute with a new tutorial please follow [these](https://github.com/amzn/emukit/tree/develop/emukit/examples) steps.\n",
     "\n",
-    "We also welcome feedback so if there is any aspect of the Emukit that we can improve please raise an issue!"
+    "We also welcome feedback so if there is any aspect of Emukit that we can improve please raise an issue!"
    ]
+  },
+  {
```
Comment (apaleyes) [inline notebooks/index.ipynb]: Empty cell

**Pull request merged**